lib/commands/doctor/checks/logged-in-user.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const execa = require('execa');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+
+const taskTitle = 'Checking logged in user';
+
+function loggedInUser() {
+    const uid = process.getuid();
+    let ghostuid;
+
+    try {
```

Comment:
I will change this to use the new `getGhostUid` fn [here](https://github.com/TryGhost/Ghost-CLI/pull/659/files#diff-e8c19ddf30935059861a1177570bd9e2R6) as soon as #659 is merged.